src/components/input/keyboardnavigation.js

```diff
@@ -159,7 +159,9 @@ function attachGamepadScript(e) {
 }
 
 // No need to check for gamepads manually at load time, the eventhandler will be fired for that
-window.addEventListener("gamepadconnected", attachGamepadScript);
+if (navigator.getGamepads) { /* eslint-disable-line compat/compat */
```

Comment:
`attachGamepadScript` above can also be placed inside.

Reply:
ESLint yells at me if I move the function inside (https://eslint.org/docs/rules/no-inner-declarations).

Reply:
Oh, right. `var foo = function (...)` should work. Or we could insert it directly as `addEventListener` callback.

Reply:
I don't think it's really necessary to get it inside of there, it'll never get executed anyway if there is no gamepad support.

If you really feel like we should, would something like this work?

```javascript
window.addEventListener("gamepadconnected", function () {
    console.log("Gamepad connected! Attaching gamepadtokey.js script");
    window.removeEventListener("gamepadconnected", this);
    require(["components/input/gamepadtokey"]);
});
```

Reply:
> would something like this work?

Nope. `this` is window. :disappointed: I didn't thought about `removeEventListener`.

OK. Leave it as it is.